style.css

```diff
@@ -334,9 +336,8 @@ body {
 .unordered_list_for_numbers {
   list-style-type: none;
   margin: 0;
-  width: 468px;
+  max-width: 78px;
```

Comment:
is this value correct? For which element is it?

Reply:
well... It appears correctly... thats box for all 3 numbers, that width came from figma when i marked a box with these numbers it appeared like this so... as far as it is correct? Should I change it?

Reply:
well then, you shouldn't set width in this case. It should be calculated based on it's content. If I ask you to add 4,5,6 next week the container wouldn't be wide enough.

Reply:
so the width in here should be 100% than?